packages/story-editor/src/components/canvas/frameElement.js

```diff
@@ -242,10 +242,15 @@ function FrameElement({ id }) {
     [handleSelectElement, id, isBackground, isSelected, speak]
   );
 
+  const { clearTransforms } = useTransform((state) => ({
+    clearTransforms: state.actions.clearTransforms,
+  }));
+
   const handleMouseDown = useCallback(
     (evt) => {
       if (!isSelected) {
         handleSelectElement(id, evt);
+        clearTransforms();
```

Comment:
This should remove all(?) issues with `isAnythingTransforming`, clearly when you do a mouse down - nothing is transforming yet - when you start dragging then we set the transforming flag.